### ISSUE TITLE
After updating to 2.0.5, the display is now vertical and has black bars on the left and right edges of the screen.

### ISSUE THREAD

**Issue by rarikuma** [MaxLastBreath/TOTK-mods#105]
It is as shown in the image below.
Is it a bug? Or am I doing something wrong with the settings?

![yuzu Early Access 4176 _ ゼルダの伝説　ティアーズ オブ ザ キングダム (64-bit) _ 1 2 1 _ NVIDIA 2024_03_17 10_36_34_result](https://github.com/MaxLastBreath/TOTK-mods/assets/135856005/b41bf664-19bb-4f9d-a311-1d0008ce76ad)

[maxlastbreath.ini.txt](https://github.com/MaxLastBreath/TOTK-mods/files/14625853/maxlastbreath.ini.txt)

**Comment by MaxLastBreath** [comment]
you got the wrong aspect ratio applied, looks like 16:10

**Comment by rarikuma** [comment]
How do I get the correct Aspect Ratio to apply?

In the TOTK Optimizer application, Aspect Ratio is selected to 16x9.

The Resolution settings in maxlastbreath.ini are as follows
Width = 3840
Height = 2160

**Comment by rarikuma** [comment]
The reason was that a folder named "Aspect Ratio 16-10" had been created in the Mod Data Location and the Aspect Ratio in yuzu's Graphic settings was set to "Force 16:10".
Deleting the "Aspect Ratio 16-10" folder and changing the Aspect Ratio in yuzu's Graphic settings to "Default (16:9)" solved the problem.

It seems that the Aspect Ratio was set to 16x10 by mistake in the first setup after updating to 2.0.5.

I then re-set it to 16x9 using the TOTK Optimizer application, but the "Aspect Ratio 16-10" folder in the Mod Data Location remained, and the "Force 16:10" setting for Aspect Ratio in the Graphic settings in yuzu setting of "Force 16:10" in yuzu's Graphic settings also remained.

I think that the TOTK Optimizer application is not processing the setting change sufficiently when changing from a setting other than 16:9 to 16:9.
I would like to see this point improved.

**Comment by MaxLastBreath** [comment]
> The reason was that a folder named "Aspect Ratio 16-10" had been created in the Mod Data Location and the Aspect Ratio in yuzu's Graphic settings was set to "Force 16:10". Deleting the "Aspect Ratio 16-10" folder and changing the Aspect Ratio in yuzu's Graphic settings to "Default (16:9)" solved the problem.
> 
> It seems that the Aspect Ratio was set to 16x10 by mistake in the first setup after updating to 2.0.5.
> 
> I then re-set it to 16x9 in the TOTK Optimizer application, but the "Aspect Ratio 16-10" folder in the Mod Data Location remained, The "Force 16:10" Aspect Ratio setting in yuzu's Graphic Settings also remained.
> 
> I think that the TOTK Optimizer application is not processing the setting change sufficiently when changing from a setting other than 16:9 to 16:9. I would like to see this point improved.

It's most likely caused by the additional 55 new aspect ratio files added yesterday, because the index of the aspect ratio changed, it also caused the AR to shift accidentally

**Comment by mckiwi11** [comment]
@rarikuma I was able to fix this myself. I changed the width in the ini file to match what is shown for “shadow resolution on the TOTK Optimizer. For me that number was “8192”. 

Then I went totk properties in yuzu and selected “stretch to fit” on aspect ration (right click on game, properties, graphics tab, aspect ration: stretch to fit). Now it looks perfect. Hope this helps. 

For context, I am on a LG C2 42” display that has a resolution in windows set to 4096x 2160.
